templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
We can also do that. What's the mimicking platforms use? Unzipped then? If so I can see having them already unzipped in the package. 

The first point still stands though - use `$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"` instead of `c:/programdata/chocolatey/lib/<pkgname>`